### PR TITLE
Avatar & badge components

### PR DESCRIPTION
**Pull request by kshitijthatte**
This PR also has some code refactor and page styling.

### CODE REVIEW

File: components/badge/bagde.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.badge-container {
+  position: relative;
+}
+
+.badge {
+  position: absolute;
+  top: -5px;
+  right: -5px;
+  color: #fff;
```
Comment (horsemaker) [inline components/badge/bagde.css]: you could use your color variable here instead.

**Review by manojasarsa — APPROVED** [review]
Looks clean! Just a small suggestion, if you can make the demo code's width according to code (inline) instead of taking the whole width of page. It will look nice.

**Review by horsemaker — APPROVED** [review]
Looks clean Kshitij! 👍
Also, the code snippet part you have implemented 👌

File: components/badge/bagde.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.badge-container {
```
Comment (poornima-alugubelly) [inline components/badge/bagde.css]: small mistake the file name is "badge.css" instead of "badge.css" here

**Review by poornima-alugubelly — APPROVED** [review]

**Review by vijaytembugade — APPROVED** [review]
Looks clean and classy. You can make sidebar fixed. Just suggestion.

**Comment by preethasri** [comment]
looks awesome kshitij

**Comment by kshitijthatte** [comment]
> Looks clean! Just a small suggestion, if you can make the demo code's width according to code (inline) instead of taking the whole width of page. It will look nice.

Thanks for the suggestion. I'll then have to also change the width of the component demo container, to make it look better I thought I'll center the code. But I couldn't work it out with hjs.

File: components/badge/bagde.css (hunk 1 above)
Comment (kshitijthatte) [reply components/badge/bagde.css]: Thanks for pointing that out Yash, I missed that.

File: components/badge/bagde.css (hunk 2 above)
Comment (kshitijthatte) [reply components/badge/bagde.css]: Thanks a lot for bringing that to my notice else I would've gone ahead with the wrong file name.

**Comment by kshitijthatte** [comment]
> Looks clean and classy. You can make sidebar fixed. Just suggestion.

Thanks Vijay, after your suggestion I tried to make it sticky so that it scrolls till it has items and then would stay fixed. Couldn't implement that yet, but I would love to have more discussion on this later after I'm done with the components.

> looks awesome kshitij

Thanks Preetha

**Pull request merged**